test/local/account_routes.js

```diff
@@ -1037,39 +1037,6 @@ test('/account/login', function (t) {
       })
     })
 
-    t.test('on for specific email', function (t) {
```

Comment:
With the abolition of the array, this test was doing nothing that is not already covered elsewhere.